### DISCUSSION TITLE
Termux dev developers in made IDE app vscode

### DISCUSSION THREAD

**Discussion by ExtinctDrift19** [termux/termux-packages#11746] [Q&A] [answered]
developers info on which IDE in made Termux

**Comment by ExtinctDrift19** [comment, marked as answer]
Termux developers what IDE you use

**Reply by sylirre** [reply]
Android Studio - for app. Nano, Neovim (no plugins) - for packages.

**Reply by ExtinctDrift19** [reply]
Termux developers what you use os windows Linux android

**Reply by sylirre** [reply]
Most of us are Linux people and use a Linux distribution installed on normal computer.
